### DISCUSSION TITLE
installing help

### DISCUSSION THREAD

**Discussion by ShayOShea** [iBotPeaches/Apktool#3518] [General]
having some troubles with installing apktool. nothing on the site is working for me

**Comment by iBotPeaches** [comment]
Unfortunately you aren't giving me much to work from here. Pick the OS - follow the steps and let me know what step you failed on, why it failed and what you tried - https://apktool.org/docs/install

**Reply by ShayOShea** [reply]
ah right sorry. Windows. stuck at step 4. i dont have a C://Windows and i have no idea what an Environment Variables System PATH variable is

**Reply by iBotPeaches** [reply]
I'd start with just using `java -jar apktool.jar` for now. Once you feel more comfortable you can work towards editing environment variables and putting binaries in windows folder (in whichever drive it exists)

**Reply by ShayOShea** [reply]
do i rename the file to that?

**Reply by iBotPeaches** [reply]
If you want - otherwise use the name you downloaded it as.

**Reply by ShayOShea** [reply]
<img width="645" alt="Screenshot 2024-02-15 133809" src="https://github.com/iBotPeaches/Apktool/assets/127950331/e715c239-40d6-4d16-9f60-e4ed9a3595ad">

this is what i get when renamed to "java -jar apktool.jar".  its not able to acces the jar or something.

**Reply by iBotPeaches** [reply]
Hi @Palmboom76 - I want to say this as nice as I can without coming off as insulting, but I believe you are far out of your league. This is a tool meant for reverse engineering binaries - if you aren't comfortable executing jar files (which is commonly done like `java -jar {jarfileHere}` then I fear this will be a tough battle for you.

I'd step back a bit from Apktool and learn some basics and feel a bit more comfortable on command line / terminal. I don't have the time to be able to help with all the basics such as this.

**Reply by ShayOShea** [reply]
Do you know any other apk decompilers?
